pennylane/interfaces/tf.py

```diff
@@ -44,25 +40,25 @@
 
 The :ref:`QNode decorator <qnode_decorator>` is the recommended way for creating QNodes
 in PennyLane. The only change required to construct a TensorFlow-capable QNode is to
-specify the ``interface='tfe'`` keyword argument:
+specify the ``interface='tf'`` keyword argument:
 
 .. code-block:: python
 
     dev = qml.device('default.qubit', wires=2)
 
-    @qml.qnode(dev, interface='tfe')
+    @qml.qnode(dev, interface='tf')
     def circuit(phi, theta):
         qml.RX(phi[0], wires=0)
         qml.RY(phi[1], wires=1)
         qml.CNOT(wires=[0, 1])
         qml.PhaseShift(theta, wires=0)
         return qml.expval(qml.PauliZ(0)), qml.expval(qml.Hadamard(1))
 
-The QNode ``circuit()`` is now a TensorFlow-capable QNode, accepting ``tfe.Variable`` objects
+The QNode ``circuit()`` is now a TensorFlow-capable QNode, accepting ``Variable`` objects
 as input, and returning ``tf.Tensor`` objects.
 
->>> phi = tfe.Variable([0.5, 0.1])
->>> theta = tfe.Variable(0.2)
+>>> phi = Variable([0.5, 0.1])
```

Comment:
same question as above

Reply:
The import paths are different depending on the version of TensorFlow:

* `tensorflow.contrib.eager.Variable`
* `tensorflow.Variable`

My feeling is that explicitly having `Variable` (assuming the user knows where to import it from) helps avoid any confusion